src/Printer.zig

```diff
@@ -3,20 +3,20 @@ const command = @import("./command.zig");
 
 const Self = @This();
 
-writer: *std.Io.Writer,
+writer: std.fs.File.Writer,
```

Comment:
Why this change? The current `Writer` is a generic writer, not bound to a file, etc.